src/main/java/dan200/computercraft/shared/computer/apis/CommandAPI.java

```diff
@@ -98,6 +99,7 @@ public void shutdown()
             }
             catch( Throwable t )
             {
+                ComputerCraft.log.error( "Error running command.", t );
```

Comment:
At the end of the day: I don't want to give players a means to programmatically spam the console. CC has been removed from modpacks in the past for violating this.